model/openapi.yml

```diff
@@ -283,6 +283,10 @@ components:
           description: The artifact download progress as a percentage.
           type: number
           format: double
+        download_rate:
```

Comment:
download_rate seems optional in agent: https://github.com/elastic/elastic-agent/blob/cc81e4b376068eff7e15330f2c9fa0969d28c8f5/internal/pkg/agent/application/upgrade/details/details.go#L38